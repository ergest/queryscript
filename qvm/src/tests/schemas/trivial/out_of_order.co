type Bar record {
    a number,
    b number,
    c number,
}
let foo = SELECT * FROM bar;
let bar [Bar] = load_json('/dev/null');
