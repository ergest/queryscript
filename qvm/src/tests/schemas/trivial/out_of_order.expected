{
    "types": {
        "let bar": [{a Float64, b Float64, c Float64}],
        "let foo": [{a Float64, b Float64, c Float64}],
        "type Bar": {a Float64, b Float64, c Float64},
    },
}